src/services/signoutService.jsx

```diff
@@ -0,0 +1,12 @@
+import { toast } from "react-hot-toast";
+const signOut = (setAuth, navigate) => {
+  localStorage.removeItem("TOKEN");
+  setAuth({
+    Authenticated: false,
+    token: "",
+  });
+  toast.error("You have logged out");
+  navigate("/");
+};
+
+export { signOut };
```

Comment:
This isn't really a service, I guess. You are just removing the item from localStorage and resetting the state. Only API calls should go inside a service file.